kubernetes/samples/scripts/create-soa-domain/domain-home-on-pv/wlst/create-domain-script.sh

```diff
@@ -5,10 +5,12 @@
 export DOMAIN_HOME=${DOMAIN_HOME_DIR}
 
 # Create the domain
+if [ -z "${JAVA_HOME}" ]; then
+  JAVA_HOME=/usr/java/latest
 wlst.sh -skipWLSModuleScanning \
```

Comment:
missing `fi` ??

Reply:
Done. Thanks!

Reply:
I'm running both internal and external Jobs.